src/components/common/helpers/Helpers.tsx

```diff
@@ -1014,16 +1014,33 @@ export const createGroupedItemsByKey = (arr: any[], key: string) => {
     }, {})
 }
 
+export const regexImageList = (imageList,serverVersion) => {
+
+    let nodeImageList = imageList.find((imageObj) => {
+        let regex = new RegExp(imageObj.groupRegex)
+        return regex.test(serverVersion)
+    })
+    
+    return nodeImageList.imageList
```

Comment:
it seems incorrect, nodeImageList will be array type and that should be returned directly

Reply:
done